packages/camlp4/camlp4.4.02.2+2/opam

```diff
@@ -0,0 +1,23 @@
+opam-version: "1"
+maintainer: "jeremie@dimino.org"
+homepage: "https://github.com/ocaml/camlp4"
+license: "LGPLv2"
+build: [
+  ["./configure" "--bindir=%{bin}%" "--libdir=%{lib}%/ocaml" "--pkgdir=%{lib}%"]
+  [make "all"]
+  [make "install" "install-META"]
+]
+depends: ["ocamlfind"]
+remove: [
+  ["rm" "-rf" "%{lib}%/camlp4"]
+  ["rm" "-f" "%{bin}%/camlp4"  "%{bin}%/camlp4boot" "%{bin}%/mkcamlp4"
+             "%{bin}%/camlp4r" "%{bin}%/camlp4rf"   "%{bin}%/camlp4orf"
+             "%{bin}%/camlp4o" "%{bin}%/camlp4of"   "%{bin}%/camlp4oof"
+             "%{bin}%/camlp4prof"  "%{bin}%/camlp4rf.opt"  "%{bin}%/camlp4orf.opt"
+             "%{bin}%/camlp4o.opt" "%{bin}%/camlp4oof.opt" "%{bin}%/camlp4r.opt"
+  ]
+]
+depexts: [
+  [ ["centos"] ["which"] ]
+]
+available: [ (!preinstalled) & (ocaml-version = "4.02.2") ]
```

Comment:
if we make this `ocaml-version >= 4.02.2 & <4.02.3` it would make development versions easier -- is it worth it?